test/integration/policy_install_operator_test.go

```diff
@@ -0,0 +1,389 @@
+			policyNoGroupYAML   = "../resources/policy_install_operator/operator_policy_no_group.yaml"
+			policyWithGroupYAML = "../resources/policy_install_operator/operator_policy_with_group.yaml"
+			policyNamePrefix    = "test-op"
+			noGroupSuffix       = "-43544"
+			withGroupSuffix     = "-43545"
+			csvName             = "quay-operator.v3.8.15"
+			subName             = "quay-operator"
+			opGroupName         = "grcqeopgroup"
+		)
+
+		var dynamicOpGroupName string
+
+		Context("When no OperatorGroup is specified", func() {
+			BeforeAll(func() {
+				_, err := common.OcManaged("create", "ns", testNS+noGroupSuffix)
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			AfterAll(func() {
+				_, err := common.OcHub(
+					"delete",
+					"-f",
+					policyNoGroupYAML,
+					"-n",
+					userNamespace,
+					"--ignore-not-found=true",
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				_, err = common.OcManaged(
+					"delete",
+					"subscription",
+					subName,
+					"-n",
+					testNS+noGroupSuffix,
+					"--ignore-not-found=true",
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				_, err = common.OcManaged(
+					"delete",
+					"operatorgroup",
+					dynamicOpGroupName,
+					"-n",
+					testNS+noGroupSuffix,
+					"--ignore-not-found=true",
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				_, err = common.OcManaged(
+					"delete",
+					"clusterserviceversion",
+					csvName, "-n",
+					testNS+noGroupSuffix,
+					"--ignore-not-found=true",
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				_, err = common.OcManaged("delete", "ns", testNS+noGroupSuffix)
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			It(policyNamePrefix+noGroupSuffix+" should be created on the hub", func() {
+				_, err := common.OcHub("apply", "-f", policyNoGroupYAML, "-n", userNamespace)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Patching the placement rule")
+				err = common.PatchPlacementRule(userNamespace, policyNamePrefix+noGroupSuffix+"-plr")
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Checking that it exists on the hub cluster")
+				rootPlc := utils.GetWithTimeout(
+					clientHubDynamic, common.GvrPolicy, policyNamePrefix+noGroupSuffix,
+					userNamespace, true, defaultTimeoutSeconds,
+				)
+				Expect(rootPlc).NotTo(BeNil())
+			})
+
+			It("operator-policy"+noGroupSuffix+" should be created on the managed cluster", func() {
+				By("Checking the policy on managed cluster in the namespace " + clusterNamespace)
+				managedPolicy := utils.GetWithTimeout(
+					clientManagedDynamic,
+					common.GvrPolicy,
+					userNamespace+"."+policyNamePrefix+noGroupSuffix,
+					clusterNamespace,
+					true,
+					defaultTimeoutSeconds,
+				)
+				Expect(managedPolicy).NotTo(BeNil())
+			})
+
+			It("operator-policy"+noGroupSuffix+" should be NonCompliant", func() {
+				By("Checking if the status of the root policy is NonCompliant")
+				Eventually(
+					common.GetComplianceState(policyNamePrefix+noGroupSuffix),
+					defaultTimeoutSeconds*2,
```

Comment:
Lots of these tests use defaultTimeoutSeconds*2. Is that necessary? It would be nice to use the default for most cases, although I understand that some of the tests here might need to wait longer for OLM to create something.